NLog.Targets.Sentry/NLog.Targets.Sentry.csproj

```
@@ -27,8 +27,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
-    <PackageReference Include="NLog" Version="4.5.11" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />

```

I think this should be left as
```suggestion
    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
```
since the version here is treated as a `>=` requirement in the package (even across major revs). Unless you're sure this is causing a problem for us, we should problem use the oldest version possible.